selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/workspace/TestWorkspaceProviderImpl.java

```diff
@@ -41,50 +39,48 @@
 @Singleton
 public class TestWorkspaceProviderImpl implements TestWorkspaceProvider {
   private static final Logger LOG = LoggerFactory.getLogger(TestWorkspaceProviderImpl.class);
+  private static final String AUTO = "auto";
 
   private final int poolSize;
-  private final ArrayBlockingQueue<TestWorkspace> testWorkspaceQueue;
-  private final ScheduledExecutorService executor;
   private final TestUser defaultUser;
   private final int defaultMemoryGb;
   private final TestWorkspaceServiceClient testWorkspaceServiceClient;
   private final TestWorkspaceServiceClientFactory testWorkspaceServiceClientFactory;
   private final WorkspaceDtoDeserializer workspaceDtoDeserializer;
 
+  private ArrayBlockingQueue<TestWorkspace> testWorkspaceQueue;
+  private ScheduledExecutorService executor;
+
   @Inject
   public TestWorkspaceProviderImpl(
+      @Named("sys.workspace_pool_size") String poolSize,
```

Comment:
Please, do not use injection with `sys.` anymore. Use injection similar to what we do in Che instead - `che.workspace_pool_size` which can be injected using system property, property in a file,  environment variable.

Reply:
This variable is kind of "**sys.threads**", so I have to do the same with threads as well, haven't it?

Reply:
Also in this case it doesn't make sense to have a parameter **--workspace-pool-size** if user be able to define the value through the system property or environment variable instead.

Reply:
It makes sense in case we will want to use selenium tests in some other way then through this script. And since it won't change much I don't see the reason to use approach that differs from Che code approach